src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -39,59 +33,94 @@ const propTypes = {
     A positive value shifts the tooltip down, and a negative value shifts it up. */
     shiftVertical: PropTypes.number.isRequired,
 
-    /** Callback to set the Ref to the Tooltip */
-    setTooltipRef: PropTypes.func.isRequired,
-
     /** Text to be shown in the tooltip */
     text: PropTypes.string.isRequired,
 
-    /** Callback to be used to calulate the width and height of tooltip */
-    measureTooltip: PropTypes.func.isRequired,
-};
+    /** Number of pixels to set max-width on tooltip  */
+    maxWidth: PropTypes.number.isRequired,
 
-const defaultProps = {};
-
-const TooltipRenderedOnPageBody = (props) => {
-    const {
-        animationStyle,
-        tooltipWrapperStyle,
-        tooltipTextStyle,
-        pointerWrapperStyle,
-        pointerStyle,
-    } = getTooltipStyles(
-        props.animation,
-        props.windowWidth,
-        props.xOffset,
-        props.yOffset,
-        props.wrapperWidth,
-        props.wrapperHeight,
-        props.tooltipWidth,
-        props.tooltipHeight,
-        props.shiftHorizontal,
-        props.shiftVertical,
-    );
-    return ReactDOM.createPortal(
-        <Animated.View
-            ref={props.setTooltipRef}
-            onLayout={props.measureTooltip}
-            style={[tooltipWrapperStyle, animationStyle]}
-        >
-            <Text style={tooltipTextStyle} numberOfLines={1}>{props.text}</Text>
-            <View style={pointerWrapperStyle}>
-                <View style={pointerStyle} />
-            </View>
-        </Animated.View>,
-        document.querySelector('body'),
-    );
+    /** Maximum number of lines to set on tooltip */
```

Comment:
Same as my other comment
```suggestion
    /** Maximum number of lines to show in tooltip */
```

Reply:
Done.